.github/workflows/screener-build.yml

```diff
@@ -272,6 +272,10 @@ jobs:
           printenv | sort ;\
           echo "SHELLOPTS $SHELLOPTS" ;\
 
+      - name: build @fluentui/react-storybook-addon
```

Comment:
need to explicitly build `react-storybook-addon` or else workflow fails in next step where stories are built.

```
Run yarn workspace @fluentui/vr-tests-react-components screener:build
yarn workspace v1.23.34
yarn run v1.23.34
$ yarn build
$ build-storybook -o dist/storybook
info @storybook/react v6.5.5
info 
info => Cleaning outputDir: /home/runner/work/fluentui/fluentui/apps/vr-tests-react-components/dist/storybook
info => Loading presets
info => Loading custom manager config
info => Compiling manager..
info => Compiling preview..
info => Loading custom manager config
ERR! Error: Cannot find module './lib/preset/preview'
ERR! Require stack:
ERR! - /home/runner/work/fluentui/fluentui/packages/react-components/react-storybook-addon/preset.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/core-common/dist/cjs/presets.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/core-common/dist/cjs/index.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/core-server/dist/cjs/index.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/core/dist/cjs/server.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/core/server.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/react/dist/cjs/server/build.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/react/bin/build.js
ERR!     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
ERR!     at Function.resolve (internal/modules/cjs/helpers.js:99:19)
ERR!     at Object.config (/home/runner/work/fluentui/fluentui/packages/react-components/react-storybook-addon/preset.js:2:29)
ERR!     at /home/runner/work/fluentui/fluentui/node_modules/@storybook/core-common/dist/cjs/presets.js:294:28
ERR!     at async _default (/home/runner/work/fluentui/fluentui/node_modules/@storybook/builder-webpack5/dist/cjs/preview/iframe-webpack.config.js:70:22)
ERR!     at async builderGeneratorFn (/home/runner/work/fluentui/fluentui/node_modules/@storybook/builder-webpack5/dist/cjs/index.js:198:16)
ERR!     at async Object.build (/home/runner/work/fluentui/fluentui/node_modules/@storybook/builder-webpack5/dist/cjs/index.js:286:14)
ERR!     at async Promise.all (index 1)
ERR!     at async buildStaticStandalone (/home/runner/work/fluentui/fluentui/node_modules/@storybook/core-server/dist/cjs/build-static.js:218:28)
ERR!     at async buildStatic (/home/runner/work/fluentui/fluentui/node_modules/@storybook/core-server/dist/cjs/build-static.js:254:5)
ERR!  Error: Cannot find module './lib/preset/preview'
ERR! Require stack:
ERR! - /home/runner/work/fluentui/fluentui/packages/react-components/react-storybook-addon/preset.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/core-common/dist/cjs/presets.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/core-common/dist/cjs/index.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/core-server/dist/cjs/index.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/core/dist/cjs/server.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/core/server.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/react/dist/cjs/server/build.js
ERR! - /home/runner/work/fluentui/fluentui/node_modules/@storybook/react/bin/build.js
ERR!     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
ERR!     at Function.resolve (internal/modules/cjs/helpers.js:99:19)
ERR!     at Object.config (/home/runner/work/fluentui/fluentui/packages/react-components/react-storybook-addon/preset.js:2:29)
ERR!     at /home/runner/work/fluentui/fluentui/node_modules/@storybook/core-common/dist/cjs/presets.js:294:28
ERR!     at async _default (/home/runner/work/fluentui/fluentui/node_modules/@storybook/builder-webpack5/dist/cjs/preview/iframe-webpack.config.js:70:22)
ERR!     at async builderGeneratorFn (/home/runner/work/fluentui/fluentui/node_modules/@storybook/builder-webpack5/dist/cjs/index.js:198:16)
ERR!     at async Object.build (/home/runner/work/fluentui/fluentui/node_modules/@storybook/builder-webpack5/dist/cjs/index.js:286:14)
ERR!     at async Promise.all (index 1)
ERR!     at async buildStaticStandalone (/home/runner/work/fluentui/fluentui/node_modules/@storybook/core-server/dist/cjs/build-static.js:218:28)
ERR!     at async buildStatic (/home/runner/work/fluentui/fluentui/node_modules/@storybook/core-server/dist/cjs/build-static.js:254:5) {
ERR!   code: 'MODULE_NOT_FOUND',
ERR!   requireStack: [
ERR!     '/home/runner/work/fluentui/fluentui/packages/react-components/react-storybook-addon/preset.js',
ERR!     '/home/runner/work/fluentui/fluentui/node_modules/@storybook/core-common/dist/cjs/presets.js',
ERR!     '/home/runner/work/fluentui/fluentui/node_modules/@storybook/core-common/dist/cjs/index.js',
ERR!     '/home/runner/work/fluentui/fluentui/node_modules/@storybook/core-server/dist/cjs/index.js',
ERR!     '/home/runner/work/fluentui/fluentui/node_modules/@storybook/core/dist/cjs/server.js',
ERR!     '/home/runner/work/fluentui/fluentui/node_modules/@storybook/core/server.js',
ERR!     '/home/runner/work/fluentui/fluentui/node_modules/@storybook/react/dist/cjs/server/build.js',
ERR!     '/home/runner/work/fluentui/fluentui/node_modules/@storybook/react/bin/build.js'
ERR!   ]
ERR! }
info => Loading presets

attention => Storybook now collects completely anonymous telemetry regarding usage.
This information is used to shape Storybook's roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://storybook.js.org/telemetry

error Command failed with exit code 1.```